libfwupdplugin/fu-udev-device.c

```diff
@@ -2241,6 +2255,8 @@ fu_udev_device_get_parent_with_subsystem(FuUdevDevice *self, const gchar *subsys
 	FuUdevDevicePrivate *priv = GET_PRIVATE(self);
 	g_autoptr(GUdevDevice) device_tmp = NULL;
 
+	if (priv->udev_device == NULL)
+		return NULL;
```

Comment:
In 2.0.0 I wonder if it's worth adding an error to this function and changing the symbol.

Reply:
Agree. Fixed.